pages/user guides/importing_to_excel.md

```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
 
 You should have one sheet or file per type of record.
 
-There are several default columns included for each record type,
-however if you have additional information you can add fields for a record and also track those.
-If this is the case you should do this before you import your data.
+There are several default columns values included for each record type, 
+however if you have additional information on your inventory that would like to include, you can create custom fields for a record (Check step 3). 
+Remember to add those fields before importing the data.
 
-## Importing your data
+## How to import your data
 
-- Select the data type you wish to import.
+1. Select the data type you wish to import.
 
-{% include image.html file="step0.PNG" %}
+    {% include image.html file="step0.PNG" %}
 
-- Select the sheet in your file that you want to import data from.
+2. Select the sheet in your file that you want to import data from.
 
-{% include image.html file="step1.PNG" %}
+    {% include image.html file="step1.PNG" %}
 
-- Map the columns of your sheet to the fields from the drop down. If you have a column that is not present as a field and you want to import it, you can add that field.
+3.  Map the columns of your sheet to the fields from the drop down. If you have a column that is not present as a field and you want to import it, you can add that field.
 
-{% include image.html file="step2.PNG" %}
+    {% include image.html file="step2.PNG" %}
 
-- Click submit to import the data.
+4. Click submit to import the data.
 
-{% include note.html content="If you are importing applications, the name field is required for each application in order to successfully import. For any of the other record types you are importing, there are no fields that are required in order for you to import them." %}
+{% include note.html content="If you are importing applications, the _name_ field is required for each application in order to successfully import. For any of the other record types you are importing, there are no fields that are required in order for you to import them." %}
 
 {% include tip.html content="The Tags option can be used to generate multiple tags and associate them with the record. You can use this track any type of categorical data. Additionally you can search and filter for records based on tags. Multiple tags can be associated based on one column of data, they are parsed and split on any commas (,)" %}
```

Comment:
this info:
>Multiple tags can be associated based on one column of data, they are parsed and split on any commas (,)

is probably useful to know all the different attributes that it is true for. ex URLs

Reply:
Do you think we should investigate ☝️ now before merging this PR. or something for us to do in the future?

Reply:
i can't see any reason it is _needed_ in order to make any of the other changes already done here.